src/app/cup/cup-cup-matches/cup-cup-matches.component.html

```diff
@@ -4,7 +4,7 @@
          <ng-container [ngTemplateOutlet]="competitionsNavigation"></ng-container>
       </div>
    </div>
-   <div class="d-none d-md-block">
+   <div class="d-none d-md-block mb-3">
```

Comment:
please add `mb-2` class to the line 2 of this file (to have a space on screens smaller than 768px)